packages/admin/admin-stories/src/admin/form/FileUpload.tsx

```diff
@@ -0,0 +1,66 @@
+import { Field, FinalFormFileUpload } from "@comet/admin";
+import { Card, CardContent, Grid } from "@mui/material";
+import { storiesOf } from "@storybook/react";
+import * as React from "react";
+import { Form } from "react-final-form";
+
+function Story() {
+    return (
+        <div style={{ width: 800 }}>
+            <Form
+                onSubmit={(values) => {
+                    //
+                }}
+                render={({ handleSubmit }) => (
+                    <form onSubmit={handleSubmit}>
+                        <Grid container spacing={4}>
+                            <Grid item xs={6}>
+                                <Card variant="outlined">
+                                    <CardContent>
+                                        <Field name="uploadDefault" label="File upload (default)" component={FinalFormFileUpload} />
+                                    </CardContent>
+                                </Card>
+                            </Grid>
+                            <Grid item xs={6}>
+                                <Card variant="outlined">
+                                    <CardContent>
+                                        <Field
+                                            name="uploadMultipleDisabled"
+                                            label="File upload (dropzone only, multiple, max file size 5 MB)"
+                                            dropzoneVariant="dropzoneOnly"
+                                            component={FinalFormFileUpload}
+                                            maxSize={5242880}
```

Comment:
Do we really want to use bytes here? 😅

Reply:
https://github.com/vivid-planet/comet/pull/1461/files/ff74d96277aa6ebb82279d574dd547ec85f9dafd..1f050af697bdb05f83c8cb807d3130cf7b2b5189#r1445819202 it was in MB before, @thomasdax98 what do you think?

Reply:
Oh, I haven't seen @thomasdax98's concern. We could at least make it more readably by using `5 * 1024 * 1024`

Reply:
Yes, I prefer the `5 * 1024 * 1024` notation better